src/wam2layers/analysis/visualization.py

```diff
@@ -35,38 +35,62 @@ def polish(ax, region):
     ax.set_ylim(region.latitude.min(), region.latitude.max())
 
 
-def _plot_precip(config, ax):
-    """Return subplot with precip."""
-    # Load config and some usful stuf.
+def _plot_input(config, ax):
+    """
+    Return subplot with input.
+    backtrack = precipitation
+    forwardtrack = evaporation
+    """
+    # Load config and some useful stuf.
     region = load_tagging_region(config)
 
     # Load data
+    start = config.tagging_start_date
+    end = config.tagging_end_date
     dates = pd.date_range(
-        start=config.preprocess_start_date,
-        end=config.preprocess_end_date,
-        freq=config.output_frequency,  # Should be output frequency, since this is used to save the data
+        start=start,
+        end=end,
+        freq=config.output_frequency,
         inclusive="left",
     )
 
-    input_files = []
-    for date in dates:
+    try:  # if backtrack data exists
+        output_files = []
+        for date in dates[0:-1]:
+            output_files.append(output_path(date, config, mode="backtrack"))
         input_files.append(input_path(date, config))
+        d_unused = xr.open_mfdataset(output_files, combine="nested", concat_dim="time")
+        ds = xr.open_mfdataset(input_files, combine="nested", concat_dim="time")
+        subset = ds.precip.sel(time=slice(start, end))
+    except:  # check if forward data exists instead
+        output_files = []
+        input_files = []
+        for date in dates[1:]:
+            output_files.append(output_path(date, config, mode="forwardtrack"))
+            input_files.append(input_path(date, config))
+        d_unused = xr.open_mfdataset(output_files, combine="nested", concat_dim="time")
+        ds = xr.open_mfdataset(input_files, combine="nested", concat_dim="time")
+        subset = ds.evap.sel(time=slice(start, end))
+    else:
+        logger.info(
+            "no output files found (so tagging input would also be meaningless)"
+        )
 
-    ds = xr.open_mfdataset(input_files, combine="nested", concat_dim="time")
-    # TODO: make region time-dependent
-    start = config.tagging_start_date
-    end = config.tagging_end_date
-    subset = ds.precip.sel(time=slice(start, end))
-    precip = (subset * region * 3600).sum("time").compute()
+    # TODO: check if backtrack files or forwardtrack files were in output
+    input = (subset * region * 3600).sum("time").compute()
 
     # Make figure
-    precip.plot(ax=ax, cmap=cm.rain, cbar_kwargs=dict(fraction=0.05, shrink=0.5))
-    ax.set_title("Cumulative precipitation during event [mm]", loc="left")
+    input.plot(ax=ax, cmap=cm.rain, cbar_kwargs=dict(fraction=0.05, shrink=0.5))
+    ax.set_title("Cumulative input during tagging [mm]", loc="left")
```

Comment:
This should cause a difference in the figures in the backtrack tests

Reply:
no, because this is the input file, so shouldn't give a plotting difference of the output (which is the only thing tested)